src/app/pages/login/login.page.ts

```diff
@@ -32,9 +34,22 @@ export class LoginPageComponent {
       .login({ username: this.user.username, password: this.user.password })
       .subscribe(
         (res: any) => {
-          this.auth.setToken(res.headers.get('Authorization'));
-          this.auth.setAuthenticated(true);
-          this.router.navigate(['home']);
+          //this.auth.setToken(res.headers.get('Authorization'));
+          //this.auth.setAuthenticated(true);
+          //this.router.navigate(['home']);
```

Comment:
Please remove these comments as they are not required anymore. Rather, adding `//CSRF` would be helpful here.

Reply:
Ok